epoxy-adapter/src/main/java/com/airbnb/epoxy/BaseEpoxyAdapter.java

```diff
@@ -140,6 +138,32 @@ protected void onModelBound(EpoxyViewHolder holder, EpoxyModel<?> model, int pos
 
   }
 
+  @CallSuper
+  @Override
+  public void onAttachedToRecyclerView(RecyclerView recycler) {
+    super.onAttachedToRecyclerView(recycler);
+
+    // set our item animator decorator so we can intercept calls to the item animator
+    // and the user doesn't have to worry about setting the decorator himself
+    if (!(recycler.getItemAnimator() instanceof ChangeAnimationDelegator)) {
```

Comment:
Should we only set this if the animator is the default item animator? Otherwise if they want to use a custom animator _without_ our wrapper we would just replace their animator. If they want to use a custom animator with our wrapper they can wrap it themselves, which I don't think is unreasonable.